All Core Devs Meetings/Meeting 123.md

```diff
@@ -0,0 +1,303 @@
+
+# All Core Devs Meeting 123
+### Meeting Date/Time: October 1, 2021, 14:00 UTC
+### Meeting Duration: 60 minutes
+### [Github Agenda](https://github.com/ethereum/pm/issues/391)
+### [Video of the meeting](https://youtu.be/-8TSQCwITA0)
+### Moderator: Tim Beiko
+### Notes: Alen (Santhosh)
+
+
+## Decisions Made / Action items
+| Decision Item | Description | Video ref |
+| ------------- | ----------- | --------- |
+|123.1 | To provide clarity to people would be to create, an upgrade, for December and name it, as Yanxi had proposed Aero glacier as a name | [16.00](https://youtu.be/-8TSQCwITA0?t=991)|
+
```

Comment:
|123.2 | Move EIP-3436 to `Last Call` | [25:30](https://youtu.be/-8TSQCwITA0?t=1530)|